app/src/modules/notifications/routes/collection.vue

```diff
@@ -8,6 +8,7 @@
 		:filter-user="filter"
 		:filter-system="roleFilter"
 		:search="search"
+		:show-select="'none'"
```

Comment:
Could remove the quotes for consistency.
```suggestion
		show-select="none"
```